tests/pulseaudio_dbus_spec.lua

```diff
@@ -202,4 +203,19 @@ b.describe("PulseAudio with DBus", function ()
 			       end
 			   end
 		   end)
+
+		   b.it("Will Cycle through all available PlaybackStreams and move them to the FallbackSink", function()
+		   	   if #core.PlaybackStreams == 0 then
+				   print("\nWARNING: Can't cycle through all available PlaybackStreams and move them to the FallbackSink because there are no PlaybackStreams in this machine")
+				   return
+			   else
+				   stream = {}
+			   	   for ps=1,#core.PlaybackStreams do
+					   stream[ps] = {}
+					   stream[ps].path = core.PlaybackStreams[ps]
+					   stream[ps].object = pulse.get_stream(connection, stream[ps].path)
+					   stream[ps].object:Move(core.FallbackSink)
```

Comment:
This test is exercising some code, but is actually not asserting that the behavior is correct. So to speak, it's not a test. You need to add assertions to make it meaningful.